frontend/src/metabase/dashboard/actions/actions.ts

```diff
@@ -71,7 +70,6 @@ export const executeRowAction = async ({
       parameters,
     });
 
-    dispatch(reloadDashboardCards());
```

Comment:
What happens here depends on where the action was triggered from (e.g. which screen), so I moved it into the component where the action is called.

Reply:
I asked questions in `frontend/src/metabase/actions/components/ActionViz/Action.tsx`. I'm not sure why this should be lifted up.